docs/guides/tenant-events.mdx

```diff
@@ -52,43 +47,30 @@ if not api_key:
 
 api_client = FlareApiClient(api_key=api_key)
 
-search_after: str | None = None
+last_from: str | None = None
 fetched_pages: int = 0
 
-while True:
-    # Rate limiting.
-    time.sleep(1)
-
-    params: dict = {
+for resp in api_client.scroll(
+    method="GET",
+    url="/firework/v2/me/feed",
+    params={
         "lite": "false",
         "order": "asc",
         "sort_by": "searchable",
+        "from": None,
     }
-    if search_after:
-        params["search_after"] = search_after
-
-    # Fetch the next page
-    resp = api_client.get(
-        "/firework/v2/me/feed",
-        params=params,
-    )
-    resp.raise_for_status()
+):
+    # Rate limiting.
+    time.sleep(1)
+
     resp_data: dict = resp.json()
 
     fetched_pages += 1
-    num_results: int = len(resp_data["items"])
+    num_results: int = len(resp_data["items"] or [])
```

Comment:
Items isn`t always present? why not use .get()?

Reply:
Linting would complain that the value passed to len was not Sized, but it seems to be fine now?